perfkitbenchmarker/vm_util.py

```diff
@@ -233,7 +238,7 @@ def _ExecuteThreadCall(target_arg_tuple, call_id, queue, parent_log_context,
     queue.put(ThreadCallResult(call_id, None, traceback.format_exc()))
 
 
-def RunParallelThreads(target_arg_tuples, max_concurrency):
+def RunParallelThreads(target_arg_tuples, max_concurrent_threads=200):
```

Comment:
This rename seems unnecessary, and I'd like to eventually merge RunParallelThreads and RunParallelProcesses if possible.